mlir/lib/Dialect/SPIRV/Transforms/UpdateVCEPass.cpp

```diff
@@ -174,6 +174,22 @@ void UpdateVCEPass::runOnOperation() {
   if (walkResult.wasInterrupted())
     return signalPassFailure();
 
+  // Update min version requirement for capabilities after deducing them
```

Comment:
```suggestion
  // Update min version requirement for capabilities after deducing them.
```
See https://llvm.org/docs/CodingStandards.html#commenting

Reply:
Done.